src/Resources/contao/drivers/DC_Table.php

```diff
@@ -6123,4 +6119,32 @@ protected function formatGroupHeader($field, $value, $mode, $row)
 
 		return $group;
 	}
+
+	/**
+	 * {@inheritdoc}
+	 */
+	public function initPicker(PickerInterface $picker)
+	{
+		$attributes = parent::initPicker($picker);
+
+		if (is_array($attributes))
```

Comment:
Check for `null` here and do an early return please. I hate these 5-level nestings 😄